packages/synapse-react-client/src/components/dataaccess/SubmissionPage.integration.test.tsx

```diff
@@ -290,8 +292,7 @@ describe('Submission Page tests', () => {
     ).not.toBeInTheDocument()
   })
 
-  // Flaky in TravisCI
```

Comment:
I ran this entire suite 50 times locally and it never failed, so I think this is no longer flaky